Dockerfile

```diff
@@ -0,0 +1,155 @@
+WORKDIR /tmp/breakpad/
+RUN git clone https://chromium.googlesource.com/linux-syscall-support
+RUN mkdir -p /tmp/breakpad/src/third_party/lss/ && mv linux-syscall-support/linux_syscall_support.h src/third_party/lss/
+WORKDIR /tmp/breakpad
+# Pull linux-syscall-support submodule
+RUN ./configure && make && \
+    make install
+# 'make install' is not provided by breakpad, so we do it manually:
+# mkdir -p /usr/local/include/breakpad && \
+# mkdir -p /usr/local/lib && \
+# cp -r src/* /usr/local/include/breakpad/ && \
+# find . -name '*.a' -exec cp {} /usr/local/lib/ \; && \
+# Cleanup
+# cd / && rm -rf /tmp/breakpad
+
+# --------------------------------------------------------
+# 4) Pull the Latest OpenTTD Source
+# --------------------------------------------------------
+WORKDIR /build
+RUN git clone --depth=1 --branch ${APPVER} https://github.com/OpenTTD/OpenTTD.git
+
+# --------------------------------------------------------
+# 5) Build OpenTTD (Release Mode)
+# --------------------------------------------------------
+WORKDIR /build/OpenTTD
+RUN mkdir build && cd build && \
+    cmake .. -DCMAKE_BUILD_TYPE=Release &&\
+    make -j"$(nproc)" && \
+    make install
+
+# --------------------------------------------------------
+# 6) Expose Ports (TCP & UDP)
+# --------------------------------------------------------
+EXPOSE 3979/tcp
+EXPOSE 3979/udp
+
+
+# ------------------------------------------------------
+# 1) Builder Stage: Compile OpenGFX
+# ------------------------------------------------------
+# FROM ubuntu:22.04 AS builder
+
+# Install system dependencies and tools for building OpenGFX
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    python3 \
+    python3-pip \
+    make \
+    imagemagick \
+    optipng grfcodec\
+    && rm -rf /var/lib/apt/lists/*
+
+# Install NML (required for compiling OpenGFX)
+RUN pip3 install nml
+
+# Clone the OpenGFX repository
+WORKDIR /build
+RUN git clone https://github.com/OpenTTD/OpenGFX.git
+
+# Compile OpenGFX via 'make'
+WORKDIR /build/OpenGFX
+RUN make
+
+
+
+# --------------------------------------------------------
+# 7) Default Command (Dedicated Server)
+# --------------------------------------------------------
+# We use 'sh -c' so that environment variables are expanded at runtime.
+# -D = Dedicated (no GUI)
+# -x <year> = Start year
+# -n 0.0.0.0:<port> = Listen on all interfaces at <port>
+#   (We combine that with the server name using a config approach or '-N' if supported.)
+#
+# In modern OpenTTD builds, there's no direct param for server_name; it typically reads openttd.cfg
+# or is set by environment variables in official Docker images. However, you could:
+#   - rely on openttd.cfg, or
+#   - pass '-N "MyServer"' if your build supports it.
+#
+# For demonstration, we show a somewhat "hybrid" approach using '-n' for net config
+# and a sample '-x' for start year. Adjust to your needs or configure openttd.cfg.
```

Comment:
`-x` does not set the start year. 

Also the documentation seems to imply things are passed, but they are not. So you're actually describing some parameters that could be manually added to the command line. In that case, make it clear that they could be passed.